spring-integration-amqp/src/main/java/org/springframework/integration/amqp/channel/PollableAmqpChannel.java

```diff
@@ -40,8 +43,8 @@
  * @author Gary Russell
  * @since 2.1
  */
-public class PollableAmqpChannel extends AbstractAmqpChannel implements PollableChannel,
-		PollableChannelManagement {
+public class PollableAmqpChannel extends AbstractAmqpChannel
+		implements PollableChannel, PollableChannelManagement, ExecutorChannelInterceptorAware {
```

Comment:
That name's no good either (a DirectChannel is a handler invoker too) - I guess we can live with it as is.